_includes/footer.html

```diff
@@ -54,6 +54,8 @@ <h3>Specifications</h3>
         <ul aria-labelledby="Specifications">
           <li><a class="page-footer__link" href="https://tc39.es/ecma262/">ECMAScript</a></li>
           <li><a class="page-footer__link" href="https://webassembly.org/">WebAssembly</a></li>
+          <li><a class="page-footer__link color-toggle">Change Color</a></li>
+          <li><a class="page-footer__link color-clear">Bring My Color Back</a></li>
```

Comment:
maybe 
```suggestion
          <li><a class="page-footer__link color-clear">Clear Colors</a></li>
```

"My color" might mean the previous color, or something else. This might make it a bit clearer.

Also, maybe we can stick some emojis in here to make it more silly. Like 🐣 for "change color" and 🥚 for "clear color"

Reply:
`Reset` or `Default` might be a better naming.